pycolmap/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.10)
+project(${SKBUILD_PROJECT_NAME} VERSION ${SKBUILD_PROJECT_VERSION})
+
+set(CMAKE_CUDA_ARCHITECTURES "native")
+if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+    # Some fixes for the Glog library.
+    add_definitions("-DGLOG_NO_ABBREVIATED_SEVERITIES")
+    add_definitions("-DGL_GLEXT_PROTOTYPES")
+    add_definitions("-DNOMINMAX")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc")
+    # Enable object level parallel builds in Visual Studio.
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /MP")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
+endif()
+
+
+find_package(colmap REQUIRED)
+
+find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)
+
+find_package(pybind11 REQUIRED)
```

Comment:
It seems like we need a minimum pybind11 version. I just tried with default packages from apt under ubuntu2204 and getting various C++ compiler errors. It should be possible to directly enforce a minimum version here in the find_package call?

Reply:
The version of pybind is pinned in the pyproject.toml and gets installed with pip when running `python -m pip install .` - did you install pycolmap in a different way?

Reply:
I was just manually configuring CMake and installing pybind through apt.

Reply:
Pinned the pybind version in CMakeLists as well to avoid this issue.